sky/build/sky_precompilation_sdk.gni

```diff
@@ -92,4 +94,30 @@ template("sky_precompilation_sdk") {
       ":$copy_config_gen_target_name",
     ]
   }
+
+  sdk_zip_gen_target_name = target_name + "_zip"
+  action(sdk_zip_gen_target_name) {
+    zip_path = "$root_out_dir/$sdk_name.zip"
+    script = "//sky/build/zip_xcodeproj.py"
+    depfile = "$target_gen_dir/$sdk_zip_gen_target_name.d"
+    outputs = [
+      depfile,
+      zip_path,
+    ]
+    args = [
+      "--depfile",
+      rebase_path(depfile, root_build_dir),
+      "--input",
+      rebase_path(sdk_dir),
```

Comment:
Can we pass the list of files to zip explicitly?  As written, this build rule won't understand its dependencies and won't work correctly in all cases.